domains/tazvn.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "MinhTaz",
+           "discord": "1155358959755071518",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.YVPjOSr_GksdKCNUdm6OzjA5Bggo3DFWl-rcDElPjIFA_VT3Nn2WjmZxObk6DYg7S3Qel_XvNynaQHlayZX5jZrIIrav5JwL6GG9qOEeGtyKSUkBoLbM1x2kOGalbpRxsLQE1uDLopG_x5moneoswztxMvTxhSRRqd6eBAqxCMKfDIFT1hGP0T3HnUI9GleoGY1qXCxyq3QucQvBOFmwN1mwp1td_pQoVuVdmXSuKaaPm6_46gIn_zDi8NFIE31LXW_IRqY2QpGYlxWNA_PojZ7Ok1ILDPkm7JxUcpr5M1djYE8q5TefOYIEKD2dQUmS-IIoAt-1MtAzTNSbuxLmNw.ZPYSg7D85E6eSECiCwYlmw.yN3dpNyxSCAOTnLHjb6w1Idj88qwmE8x-g_zTBrF97FsBCBDKNVd2mmOhrBI4T8dyHsz9vuwqYl5BNiJwMC7TVIYzkqC1xxjg0nCErJkUWw.8j4DZvyx73y5ZUnwoFDW3w"
+        },
+    
+        "record": {
+            "CNAME": "https://minhtaz.github.io/"
```

`CNAME` cannot contain `/` or `https://`
```suggestion
            "CNAME": "minhtaz.github.io"
```

---

what @21Z said